### PR TITLE
fallback into english if no localized news, fix incorrect unread count due to missed featured news

### PR DESCRIPTION
**Pull request by ice-cronus**
1. For now if app requests news per-language and localized new is missed (no translation), that not localized news entry is not returned at all. This PR changes that and default english one is retunted in case of missing localized entry.
2. Featured news were calculated in unread news count, even if they were replaced with a new one, as a result if user missed some featured news until they are replaced - GetUnreadNewsCount returns that missed news count all the time. This PR fixes that as well, so for the featured one only 1 or 0 unread count is possible, due to only one actual featured news entry

### CODE REVIEW

**Comment by ice-cronus** [comment]
TODO: rebase with master when #34 would be merged

File: news/news_get.go [hunk 1]
```diff
@@ -53,21 +70,46 @@ func (r *repository) GetNews(ctx context.Context, newsType Type, language string
...
+						LEFT JOIN news_viewed_by_users nvu_en
+							ON nvu_en.language = n_en.language
+							AND nvu_en.news_id = n_en.id 
+							AND nvu_en.user_id = $1
+				WHERE n_en.language = '%[1]v'
+					AND (n_en.type = $3)
+					AND (n_en.created_at >= $5 OR n.created_at >= $5)
+					AND nvu_en.created_at IS NULL 
+				    AND nvu.created_at IS NULL
+			) regular_count, featured_count`, fallbackLanguage)
```
Comment (ice-ares) [inline news/news_get.go]: Thats a cross join. U're gonna have 2 rows in the result. Join them on something to have 1 row

File: news/news_get.go [hunk 2]
```diff
@@ -53,21 +70,46 @@ func (r *repository) GetNews(ctx context.Context, newsType Type, language string
...
+//nolint:funlen // SQL
 func (r *repository) GetUnreadNewsCount(ctx context.Context, language string, createdAfter *time.Time) (*UnreadNewsCount, error) {
 	if ctx.Err() != nil {
 		return nil, errors.Wrap(ctx.Err(), "context failed")
 	}
 	args := []any{requestingUserID(ctx), language, RegularNewsType, FeaturedNewsType, createdAfter.Time}
-	sql := `SELECT COALESCE(COUNT(n.id), 0) as count
-				FROM news n	
+	sql := fmt.Sprintf(`
+		WITH featured_count AS (
```
Comment (ice-ares) [inline news/news_get.go:80]: U're not taking en into account

File: news/news_get.go [hunk 3]
```diff
@@ -53,21 +70,46 @@ func (r *repository) GetNews(ctx context.Context, newsType Type, language string
...
+					AND nvu.user_id = $1
+			WHERE 
+				n.language = $2 
+				AND n.type = $4 
+				AND n.created_at >= $5 
+			ORDER BY n.created_at DESC LIMIT 1
+		) 
+		SELECT featured_count.count + regular_count.count as count FROM 
+		(
+			SELECT COALESCE(COUNT(*), 0) as count
```
Comment (ice-ares) [inline news/news_get.go]: U should count(coalesce(n.id,n_en.id))

**Pull request merged**